### PR TITLE
OCPBUGS-31434#adding IngressController update

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions:
4.16+

Issue:
https://issues.redhat.com/browse/OCPBUGS-31434

Link to docs preview:
https://79209--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/overview/installing-preparing.html#installing-preparing-existing-components

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-process.adoc [hunk 1]
```diff
@@ -72,6 +72,8 @@ If possible, use this feature to avoid having to provision and maintain the clus
 
 With installer-provisioned infrastructure clusters, {product-title} manages all aspects of the cluster, including the operating system itself. Each machine boots with a configuration that references resources hosted in the cluster that it joins. This configuration allows the cluster to manage itself as updates are applied.
 
+For some installer-provisioned infrastructure installations, for example vSphere and bare metal, the external traffic that reaches the `ingressVIPs` is not balanced between the default `IngressController` replicas. For vSphere and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController`, or router, performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas.
```
Comment (bscott-rh) [inline modules/installation-process.adoc:75]: I don't think the information really belongs in this section. the rest of the "The installation process with installer-provisioned infrastructure" sub-section doesn't really deal with specifics.

File: installing/installing-preparing.adoc [hunk 2]
```diff
@@ -62,6 +62,7 @@ For installer-provisioned infrastructure installations, you can use an existing
...
 
+For some installer-provisioned infrastructure installations, for example vSphere and bare metal, the external traffic that reaches the `ingressVIPs` is not balanced between the default `IngressController` replicas. For vSphere and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController`, or router, performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas. For more information about the baseline `IngressController` performance, see xref:../scalability_and_performance/optimization/routing-optimization.adoc#baseline-router-performance_routing-optimization[Baseline Ingress Controller (router) performance]. For more information about configuring an external load balancer, see...
```
Comment (bscott-rh) [inline installing/installing-preparing.adoc:65]: looks like you're missing a link at the end of this line.

Comment (bscott-rh) [inline installing/installing-preparing.adoc:65]: I think the content fits well here.

Comment (bscott-rh) [inline installing/installing-preparing.adoc:65]:
```suggestion
For some installer-provisioned infrastructure installations, for example on the {vmw-first} and bare metal platforms, the external traffic that reaches the ingress virtual IP is not balanced between the default `IngressController` replicas. For {vmw-short} and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController` router performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas. For more information about the baseline `IngressController` performance, see xref:../scalability_and_performance/optimization/routing-optimization.adoc#baseline-router-performance_routing-optimization[Baseline Ingress Controller (router) performance]. For more information about configuring an external load balancer, see...
```

rewording suggestions

**Comment by brendan-daly-red-hat** [comment]
@gpei, PTAL at this draft update.

File: installing/installing-preparing.adoc [hunk 3]
```diff
@@ -62,6 +62,7 @@ For installer-provisioned infrastructure installations, you can use an existing
...
 
+For some installer-provisioned infrastructure installations, for example on the {vmw-first} and bare metal platforms, the external traffic that reaches the ingress virtual IP is not balanced between the default `IngressController` replicas. For {vmw-short}  and bare metal installer-provisioned infrastructure installations where it is required to exceed the baseline `IngressController` router performance, you must configure an external load balancer. Configuring an external load balancer achieves the performance of multiple `IngressController` replicas. For more information about the baseline `IngressController` performance, see xref:../scalability_and_performance/optimization/routing-optimization.adoc#baseline-router-performance_routing-optimization[Baseline Ingress Controller (router) performance]. For more information about configuring an external load balancer, see xref:../installing//installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#nw-osp-configuring-external-load-balancer_ipi-install-installation-workflow[Configuring a user-managed load balancer].
```
Comment (dfitzmau) [inline installing/installing-preparing.adoc:65]: 1. ingress virtual IP > ingress virtual IP (VIP)
2. where it is required to exceed the baseline `IngressController` router performance > where exceeding the baseline `IngressController` router performance is expected

Comment (dfitzmau) [reply installing/installing-preparing.adoc:65]: /remove-label peer-review-in-progress

/label peer-review-done

**Pull request merged**